src/vs/workbench/contrib/terminalContrib/suggest/common/terminalSuggestConfiguration.ts

```diff
@@ -201,8 +201,10 @@ export function registerTerminalSuggestProvidersConfiguration(availableProviders
 	const providersProperties: IStringDictionary<IConfigurationPropertySchema> = {};
 
 	const corePwshProviderId = 'core:pwsh-shell-integration';
+	const lspProviderId = 'lsp';
```

Comment:
I think we might want to attach provider unique id (language extension's debugDisplayName) after `lsp`

Then, we can set default true on insiders for all of the providers that start with lsp.

Reply:
How about `lsp:<extensionId>` instead?